galpy/util/coords.py

```diff
@@ -900,6 +991,119 @@ def cov_dvrpmllbb_to_vxyz_single(d,e_d,e_vr,pmll,pmbb,cov_pmllbb,l,b):
                  [-numpy.cos(l)*numpy.sin(b),-numpy.sin(l)*numpy.sin(b), numpy.cos(b)]])
     return numpy.dot(R.T,numpy.dot(cov_vrvlvb,R))
 
+def cov_vxyz_to_galcencyl(cov_vxyz, phi, Xsun=1., Zsun=0.):
+    """
+    NAME:
+       cov_vxyz_to_galcencyl
+    PURPOSE:
+       propagate uncertainties in vxyz to galactocentric cylindrical coordinates
+    INPUT:
+       cov_vxyz - uncertainty covariance in U,V,W
+       phi - angular position of star in galactocentric cylindrical coords
+    OUTPUT:
+       cov(vR,vT,vz) [3,3]
+    HISTORY:
+       2018-03-22 - Written - Mackereth (LJMU)
+       2020-09-21- Moved to coords.py - Mackereth (UofT)
+    """
```

Comment:
This is a new function? If so, could you add it to the API docs in `docs/source/reference/coords.rst` similar to other functions?
None of these following new functions are hit by any tests, see the report [here](https://codecov.io/gh/jobovy/galpy/pull/434/src/galpy/util/coords.py?before=galpy/util/coords.py#L1009).
Also same comment re:single vs. array as before...